ToDoApp/Extensions/UITextFieldExtension.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UITextFieldExtension.swift
+//  ToDoApp
+//
+//  Created by deniss.lobacs on 24/02/2022.
+//
+
+import UIKit
+
+extension UITextField {
+    static func customTextField(_ placeholder: String) -> UITextField {
```

Comment:
it's classic situation, when we need to create heir of class, create pls class CustomTextField: UITextField {

and setup all needed parameters